applications/nrf5340_audio/src/bluetooth/bt_stream/bt_le_audio_tx/bt_le_audio_tx.c

```diff
@@ -53,7 +53,7 @@ static bool initialized;
 static struct bt_le_audio_tx_info tx_info[SRC_STREAM_COUNT];
 
```

Comment:
I would suggest we need to add some doc for this static function, since it is an important function and also the mechanism is complex.